op-service/eth/blob_test.go

```diff
@@ -1,7 +1,10 @@
 package eth
 
 import (
+	"math/rand"
 	"testing"
+
+	"github.com/stretchr/testify/require"
 )
 
```

Comment:
It would be nice to have some fuzz roundtrip tests for encode/decode

Reply:
added